### PR TITLE
more cleanup to synopsysctl messages

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/423#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_describe.go [hunk 1]
```diff
@@ -72,14 +72,14 @@ var describeAlertCmd = &cobra.Command{
 // describeBlackDuckCmd Show details of one or many Black Ducks
 var describeBlackDuckCmd = &cobra.Command{
 	Use:     "blackduck [namespace]...",
-	Example: "synopsysctl describe blackducks\nsynopsysctl describe blackduck bdnamespace\nnsynopsysctl describe blackducks bdnamespace1 bdnamespace2",
+	Example: "synopsysctl describe blackducks\nsynopsysctl describe blackduck <namespace>\nnsynopsysctl describe blackducks <namespace1> <namespace2>",
```
Comment (rrati) [inline pkg/synopsysctl/cmd_describe.go]: no s on blackduck

File: pkg/synopsysctl/cmd_describe.go [hunk 2]
```diff
@@ -101,14 +101,14 @@ var describeBlackDuckCmd = &cobra.Command{
 // describeOpsSightCmd Show details of one or many OpsSights
 var describeOpsSightCmd = &cobra.Command{
 	Use:     "opssight [namespace]...",
-	Example: "synopsysctl describe opssights\nsynopsysctl describe opssight opsnamespace\nsynopsysctl describe opssights opsnamespace1 opsnamespace2",
+	Example: "synopsysctl describe opssights\nsynopsysctl describe opssight <namespace>\nsynopsysctl describe opssights <namespace1> <namespace2>",
```
Comment (rrati) [inline pkg/synopsysctl/cmd_describe.go]: no s on opssight

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/423#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/423#pullrequestreview-248320519" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@mphammer: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/423#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/423#pullrequestreview-250618009" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**